src/main/kotlin/org/malachite/estella/offer/domain/OfferPayloads.kt

```diff
@@ -45,14 +46,29 @@ data class Skill(
 data class OfferResponse(
     val id: Int?, val name: String, val description: String, val position: String,
     val minSalary: Long, val maxSalary: Long, val localization: String,
-    val creatorId: Int, val skills: Set<DesiredSkill>
+    val organization: OrganizationResponse, val skills: Set<DesiredSkill>
 ) {
     companion object {
         fun fromOffer(offer: Offer): OfferResponse {
             return OfferResponse(
-                offer.id, offer.name, offer.description.characterStream.readText(),
-                offer.position, offer.minSalary, offer.maxSalary, offer.localization, offer.creator.id!!, offer.skills
+                offer.id,
+                offer.name,
+                offer.description.characterStream.readText(),
+                offer.position,
+                offer.minSalary,
+                offer.maxSalary,
+                offer.localization,
+                offer.getOrganizationResponse(),
+                offer.skills
             )
         }
     }
-}
\ No newline at end of file
+}
+
+data class OrganizationResponse(
+    val name: String
+)
+
+fun Offer.getOrganizationResponse() = OrganizationResponse(
```

Comment:
Nie wiem czy nie lepsze byłoby samo getOrganizationName. Bo jest jakieś zastosowanie tego organizationResponse zamiast samego stringa?

Reply:
jeszcze nie, ale zostawiłem furtkę na przyszłość, jakbyśmy chcieli coś więcej mieć, to będzie łatwiej dodać, niż później kombinować

Reply:
Z moich ust to będzie zabawnie brzmiało ale przedwczesna optymalizacja to źródło wszelkiego zła XDD

Reply:
Niby tak, ale still XD

Reply:
Według mnie, to nawet lepiej wygląda - tak trochę bardziej JSowo - masz obiekt w obiekcie, więc wiesz, do czego to name należy. Jest to nawet trochę bardziej opisowe, więc według mnie jest git XD

Reply:
Nie wiem czy JS jest czymś na czym powinniśmy się wzorować, więc skoro tobie to pomaga to leci approve XD